src/payment/payment.ts

```diff
@@ -69,3 +70,16 @@ export interface ThreeDSecureToken {
 export interface HostedInstrument {
     shouldSaveInstrument?: boolean;
 }
+
+export interface PaypalInstrument {
+    vault_payment_instrument: boolean | null;
```

Comment:
How come it's `boolean | null` instead of just `?`

Reply:
I'm pretty sure big-pay requires values to be either defined or null. I'm not 100% sure, that's at least the pattern big-pay client follows.

We can in the future try to work out which values must actually be `null` and which ones can just be `?: <type>`.